ui-core/src/main/java/com/adyen/checkout/ui/core/internal/ui/model/AddressOutputData.kt

```diff
@@ -34,15 +35,20 @@ data class AddressOutputData(
             city.validation.isValid() &&
             country.validation.isValid()
 
-    override fun toString(): String {
+    fun getDisplayAddress(locale: Locale): String {
         return listOf(
             street.value,
             houseNumberOrName.value,
             apartmentSuite.value,
             postalCode.value,
             city.value,
             stateOrProvince.value,
-            country.value,
+            getDisplayCountry(country.value, locale),
         ).filter { it.isNotBlank() }.joinToString(" ")
     }
+
+    private fun getDisplayCountry(isoCode: String, locale: Locale): String {
```

Comment:
Is it possible to use `CountryUtils.getCountryName()`, instead of adding this function here?